plugins/out_kinesis_firehose/firehose_api.c

```diff
@@ -165,6 +165,7 @@ static int process_event(struct flb_firehose *ctx, struct flush *buf,
     size_t len;
     size_t tmp_size;
     void *compressed_tmp_buf;
+    char *out_buff;
```

Comment:
nit: could you change all out_buff to out_buf? There are 31 "buff" occurrences  in the codebase but 3290 occurrences of buf. Buf is the common abbreviation for a buffer

Reply:
ok